oauth2-useragent-core/src/main/java/com/microsoft/alm/oauth2/useragent/JavaFxProvider.java

```diff
@@ -14,7 +14,8 @@
     private static final List<String> REQUIREMENTS = Collections.unmodifiableList(Arrays.asList(
         "Oracle Java SE 7 update 6 or higher, Oracle Java SE 8, OR OpenJDK 8.",
         "JavaFX or OpenJFX runtime JAR.",
-        "A desktop environment."
+        "A desktop environment.",
+        "Oracle Java SE 7 is not supported on Mac OS X 10.11 and greater.  Please upgrade to Java 8."
     ));
     private final File[] potentialJavaFxJarLocations = new File[]{

```

Comment:
Excellent observation!  I checked and on both Windows and Linux with Java 8 and it was doing what I intended.  I nevertheless added commit ea2e44e245b50150f5e28fa1fee558c540e5731c to make sure the paths are relative, in case the documented behaviour decides to manifests itself somewhere. :)